sdk/identity/Azure.Identity/src/DefaultAzureCredential.cs

```diff
@@ -8,15 +8,15 @@
 
 namespace Azure.Identity
 {
-    public class SystemCredential : AggregateCredential
+    public class DefaultAzureCredential : ChainedTokenCredential
```

Comment:
I am not thrilled about the name. If we ever decide that we want a different credential type to be the default, this name will be misleading. And I can totally imagine the recommended credential will change over time.

Reply:
The upside is that the credentials that are chained within `DefaultAzureCredential` can be changed over time and could even be set to a single credential if necessary.

Reply:
I don't think we can change this arbitrarily. I think we can append new items to the chain. We cannot remove or rearrange the chain. This would be a breaking change.